lib/date_range_form_field.dart

```diff
@@ -223,4 +221,9 @@ class DateRangeField extends FormField<DateTimeRange> {
   /// Any valid format from the intl package is usable.
   /// If null, this will default to 'MM/dd/yyyy'.
   final DateFormat? dateFormat;
+
+  /// This is the text style of the field when the hint text is equal to [null].
+  ///
+  /// If null, this will default to ([DefaultTextStyle]).
```

Comment:
It looks like this will style the both the hint text and the date range which is displayed, so the documentation here should reflect that.

Reply:
You are right, I'm gonna review this as soon as possibile

Reply:
@Cali0707 with https://github.com/JMAConsulting/date_range_form_field/pull/13/commits/a7ce4f0dea9d5bc441677d4d1a0b085552caa5ca should be fine, I'm open to suggestions too. Let me know  👀

Reply:
This looks better, however from looking at your code I believe that this is the default text style of the field, but will be overridden if there is a ```hintStyle``` provided and the ```hintText``` is being displayed. So maybe we could say something along the lines of:
```dart
/// This is the default [TextStyle] of the field.
///
/// If a hintStyle is provided in the decoration, then the hintStyle will be applied when the hintText is displayed.
/// If null, this will default to ([DefaultTextStyle])
```